### PR TITLE
[WIP] number instance & error Proxy/API

### PR DESCRIPTION
**Pull request by QuanticPotatoes**
- Remove instance's number in URI.
- Catch error from proxy & API while execute scroll request.

### CODE REVIEW

File: src/api/exporters/ezsLocals/JSONLDObject.js [hunk 1]
```diff
@@ -61,15 +61,28 @@ function mergeSimpleField(output, field, data) {
...
 
     return uri;
 }
 
+function getUri(uri) {
+    const u = removeNumberInstance(uri);
+
+    if (u.indexOf('http://') !== 0 &&
+        u.indexOf('https://') !== 0) {
+        return path.normalize(this.getParam('hostname', 'http://lod.istex.fr/').concat(u));
```
Comment (touv) [inline src/api/exporters/ezsLocals/JSONLDObject.js]: lod.istex.fr should be define in the config file isn't it ?

Comment (parmentf) [reply src/api/exporters/ezsLocals/JSONLDObject.js]: It seems that `this` is undefined within that function.

Comment (parmentf) [reply src/api/exporters/ezsLocals/JSONLDObject.js]: @touv Why do you want to append `http://lod.istex.fr/` to any URI?
It should be what is called `host` (`http://wos-category.lod.istex.fr/`) in [default.js](https://github.com/Inist-CNRS/lodex/blob/master/config/default.js#L10), shouldn't it?

Besides, in what cases `u` does not begin with `http`?
default.js set a default value to `host`, beginning with `http`.

Comment (touv) [reply src/api/exporters/ezsLocals/JSONLDObject.js]: now, "this" is undefined, before "this" was the ezs scope.  this.getParam('hostname') is used to fetch ezs statement parameter (https://github.com/touv/node-ezs#scope)

of course, we should use "host" from defaut.js, and to get it, one way is to use ezs parameter like this https://github.com/Inist-CNRS/lodex/blob/master/src/api/exporters/exportJsonld.js#L14

**Pull request merged**